addons/medical/functions/fnc_getBloodLoss.sqf

```diff
@@ -0,0 +1,22 @@
+#include "..\script_component.hpp"
+/*
+ * Author: DartRuffian
+ * Calculate the total blood loss of a unit.
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ *
+ * Return Value:
+ * Total blood loss of unit (litres/second) <NUMBER>
+ *
+ * Example:
+ * player call ace_medical_fnc_getBloodLoss
+ *
+ * Public: Yes
+ */
+
+params [["_unit", objNull, [objNull]]];
+
+if (!local _unit) exitWith { ERROR_1("unit [%1] is not local",_unit) };
```

Comment:
Needs to return a number since header mandates so (`-1` for invalid would be a decent choice imo).